extensions/ql-vscode/src/cli.ts

```diff
@@ -970,6 +970,12 @@ export class CodeQLCliServer implements Disposable {
     }
   }
 
+  async packResolveDependencies(dir: string, mode: 'use-lock'): Promise<{ [pack: string]: string }> {
```

Comment:
I think this is what you want:

```suggestion
  async packResolveDependencies(dir: string, mode = 'use-lock'): Promise<{ [pack: string]: string }> {
```